data/Med16-11.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Medh",
+  "type": "personal",
+  "bio": "**A Nerd**",
+
+  "tags": ["tag1", "tag2", "tag3"],
```

Comment:
Hey, you are missing adding tags here. You can take reference from here - https://linkfree.io/docs/how-to-guides/tags.

**Note: This is not a blocker in merging this PR. Just a suggestion**